src/routes/comment.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+
+const commentRouter = express.Router();
+
+const {
+  addComments,
+  getCommentsByPost,
+  getCommentsById,
+  updateComments,
+  deleteComments,
+} = require('../controller/comments');
+const authUser = require('../middleware/authenticatedUser');
+
+commentRouter.get('/post/:id/comments', getCommentsByPost);
+commentRouter.get('/comment/:id', getCommentsById);
+
+commentRouter.post('/comment', authUser, addComments);
+commentRouter.put('/comment/:id', authUser, updateComments);
+commentRouter.delete('/comment/:id', authUser, deleteComments);
```

Comment:
Instead of passing the middleware for each route you can use it like this:
```js
app.use(authUser);
//Here you can set all protected routes
commentRouter.post('/comment', addComments);
commentRouter.put('/comment/:id', updateComments);
commentRouter.delete('/comment/:id', deleteComments);
```